app/controllers/foo_controller.rb

```diff
@@ -3,6 +3,7 @@ def new_user
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[staging][rubocop]__
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

<!-- Comment by [SideCI](https://sideci.com) -->